micro-rdk/src/common/webrtc/candidates.rs

```diff
@@ -387,12 +387,14 @@ impl CandidatePair {
             CandidatePairState::InProgress | CandidatePairState::Succeeded => {
                 if let Some(req) = self.current_binding_request.as_mut() {
                     // Retry while pair is InProgress, Ta is set a 500ms.
-                    if now - req.req_time < Duration::from_millis(250) {
+                    if now - req.req_time < Duration::from_millis(500) {
                         return None;
                     }
-                    self.binding_req_sent += 1;
-                    req.req_time = now;
-                    return Some(req.id);
+                    if !req.resp_recv {
```

Comment:
Just double checking that this is intended, but if this branch is not taken you will continue to line 401 since there is no `else` case.

Reply:
yup we create a new request if the preceding one was successful